assignments/prototypes.js

```diff
@@ -32,7 +52,23 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+
+function Humanoid(myObj) {
+  CharacterStats.call(this, myObj)
+  this.faction = myObj.faction
```

Comment:
While this behavior does right itself when typed in the browser console. I wouldn't rely on it myself.

Reply:
As a safe no semi-colon strategy, if you can feel out where they will right themselves and not. If its longer or more complicated than a single statement on the right side of the `=`, that can be assigned it will break. For instance if `b` became a function and we were relying on its return for our assignment value.